semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/Scalacp.scala

```diff
@@ -9,7 +9,7 @@ import scala.meta.internal.semanticdb.SymbolInformation.{Kind => k}
 import scala.meta.internal.semanticdb.Scala._
 import scala.tools.scalap.scalax.rules.scalasig._
 
-class Scalacp private (classfile: ToplevelClassfile)
+class Scalacp private (val classfile: ToplevelClassfile)
```

Comment:
Why does this need to become a val?

Reply:
Looks like that will no longer be necessary, once lookup becomes a standalone member of Scalacp.

Reply:
We still need access to `settings` and `index`, do you really think it's worth writing forwarder methods in Scalacp?